lib/components/user/saved-trip-wizard.js

```diff
@@ -27,11 +28,15 @@ const SavedTripWizard = ({ monitoredTrip, onComplete, panes }) => {
   }
 
   return (
-    <SequentialPaneDisplay
-      initialPaneId='basics'
-      onComplete={onComplete}
-      paneSequence={paneSequence}
-    />
+    <div>
+      {/* TODO: Improve navigation. */}
+      <p><LinkButton to='/'>Back to Trip Planner</LinkButton></p>
```

Comment:
Pretty much everywhere the LinkButton is used, it is surrounded with a `p` or `span` tag. It might be better just to just use styled-components or pass standard style/className props instead. I don't think this should be a blocking change, but I do think we will eventually need to address it.

Reply:
As indicated in TODO, let's discuss a layout in which we can fit this button and other relevant commands.